test/DynamoCoreWpfTests/CoreUITests.cs

```diff
@@ -468,6 +468,32 @@ public void PreferenceSetting_BackgroundPreview_1_0API()
             #endregion
         }
 
+        [Test, RequiresSTA]
+        [Category("DynamoUI")]
+        public void PreferenceSetting_RenderPrecision()
```

Comment:
could you test what happens if we encounter an old settings file that does not include the render precision property?

Reply:
Same

Reply:
No problem